src/mono/wasm/runtime/workers/dotnet-crypto-worker.ts

```diff
@@ -3,7 +3,7 @@
 
 import { setup_proxy_console } from "../debug";
 import type { InitCryptoMessageData } from "../crypto-worker";
-import type { MonoConfig } from "../types";
+// import type { MonoConfig } from "../types";
```

Comment:
Can we delete commented out code?

Reply:
Sorry Eric, this is just PR to be able to run CI tests. I will give you link to actual clean fix PR soon